### ISSUE TITLE
@babel/preset-react and babel-plugin-dynamic-import-node

### ISSUE THREAD

**Issue by KatSick** [happo/happo.io#257]
is there any reason why `@babel/preset-react` and `babel-plugin-dynamic-import-node` in `dependencies` section of the package.json and not in `devDependencies`?

It bloats our application node_modules a lot, because of it

**Comment by ljharb** [comment]
Because they're runtime dependencies:
 - https://github.com/happo/happo.io/blob/c4eb30a0e74a14da8aba307ef10f6c04327caf12/src/createWebpackBundle.js#L34
 - https://github.com/happo/happo.io/blob/c4eb30a0e74a14da8aba307ef10f6c04327caf12/src/createWebpackBundle.js#L5

**Comment by KatSick** [comment]
@ljharb I agree, but for example here https://github.com/happo/happo.io/blob/c4eb30a0e74a14da8aba307ef10f6c04327caf12/src/createWebpackBundle.js#L10 you are using webpack but do not specify it in dependencies

I'm asking, because we are using Happo with Cypress, without any babel configurations, and I'm wondering how can I avoid it.

as I understand, as soon as this is not a monorepo with more modular packages which can be published separately it is not possible?

--
thanks for super fast reponse!

**Comment by ljharb** [comment]
https://github.com/happo/happo.io/blob/master/package.json#L109 is where webpack is specified as a peer dep.

I don't think you can avoid it.

**Comment by KatSick** [comment]
@ljharb can we move babel presets to peerDeps as well? (sorry for all the questions)

**Comment by ljharb** [comment]
Peer deps are strictly required, so that wouldn't help your problem at all - it'd just be a breaking change and additional burden on users to install them themselves.
